lib/osf-components/addon/components/subjects/manager/component.ts

```diff
@@ -114,6 +114,7 @@ export default class SubjectManagerComponent extends Component {
 
         try {
             yield this.model.updateM2MRelationship('subjects', selectedSubjects);
```

Comment:
I wonder why `updateM2MRelationship ` doesn't push the response data to the store and return the liveArray??

Reply:
`updateM2MRelationship` patches to the relationship link of the relationship. So that means the returned response will be the information of the relationship, not the model itself. Therefore cannot really push the response data into the store to update the model.

Reply:
That makes sense. When we patch through the resource, we get the updated model as response.